huaweicloud/services/rds/resource_huaweicloud_rds_upgrading_minor_version.go

```diff
@@ -0,0 +1,124 @@
+package rds
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var dbKernelUpdateNonUpdatableParams = []string{"instance_id"}
+
+// @API RDS POST /v3/{project_id}/instances/{instance_id}/db-upgrade
+func ResourceRdsUpgradingMinorVersion() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceRdsUpgradingMinorVersionCreate,
+		ReadContext:   resourceRdsUpgradingMinorVersionRead,
+		UpdateContext: resourceRdsUpgradingMinorVersionUpdate,
+		DeleteContext: resourceRdsUpgradingMinorVersionDelete,
+
+		CustomizeDiff: config.FlexibleForceNew(dbKernelUpdateNonUpdatableParams),
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"is_delayed": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"job_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"enable_force_new": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"true", "false"}, false),
+				Description:  utils.SchemaDesc("", utils.SchemaDescInput{Internal: true}),
+			},
+		},
+	}
+}
+
+func resourceRdsUpgradingMinorVersionCreate(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	const (
+		httpUrl = "v3/{project_id}/instances/{instance_id}/db-upgrade"
+		product = "rds"
+	)
+
+	instanceID := d.Get("instance_id").(string)
+
+	client, err := cfg.NewServiceClient(product, region)
+	if err != nil {
+		return diag.Errorf("error creating RDS client: %s", err)
+	}
+
+	url := client.Endpoint + httpUrl
+	url = strings.ReplaceAll(url, "{project_id}", client.ProjectID)
+	url = strings.ReplaceAll(url, "{instance_id}", instanceID)
+
+	body := map[string]interface{}{}
+	if v, ok := d.GetOk("is_delayed"); ok {
+		body["is_delayed"] = v.(bool)
+	}
```

Comment:
```suggestion
	body := map[string]interface{}{
	  "is_delayed" : d.Get("is_delayed")
	}
```
and use a separate method to construct parameters